x/feegrant/module.go

```diff
@@ -0,0 +1,147 @@
+package feegrant
+
+import (
+	"encoding/json"
+
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+var (
+	_ module.AppModule      = AppModule{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
+
+// AppModuleBasic defines the basic application module used by the feegrant module.
+type AppModuleBasic struct{}
+
+// Name returns the feegrant module's name.
+func (AppModuleBasic) Name() string {
+	return ModuleName
+}
+
+// RegisterCodec registers the feegrant module's types for the given codec.
+func (AppModuleBasic) RegisterCodec(cdc *codec.Codec) {
+	RegisterCodec(cdc)
+}
+
+// DefaultGenesis returns default genesis state as raw bytes for the feegrant
+// module.
+func (AppModuleBasic) DefaultGenesis() json.RawMessage {
+	return []byte("[]")
```

Comment:
```suggestion
	return []byte("{}")
```

Reply:
You made this comment before and I responded:
https://github.com/cosmos/cosmos-sdk/pull/5207#discussion_r343323025